### ISSUE TITLE
Comparing this dataset to the older dump from IA

### ISSUE THREAD

**Issue by jneen** [PretendoNetwork/archival-tools#8]
Hey there!

Congrats on finishing the archive. If it's not too much trouble, I'd like to take the time to compare the course set from this archive to the older one by [JustAnotherArchivist](https://archive.org/details/@justanotherarchivist) on [Internet Archive](https://archive.org/details/super_mario_maker_courses_202105). It's likely that since courses were still being deleted in the interim, that archive has courses that wouldn't be captured now. I have a metadata db based on the json there and would be happy to run a comparison.

Thanks!
jneen

**Comment by cheater** [comment]
agreed, there's also [this](https://archive.org/download/smm_levels/). both dumps will probably contain courses that have been deleted, so it's important to check whether there are any that are gone in the latest dump. thank you! 🙏
